benchmarks/README.md

```diff
@@ -1,30 +1,105 @@
 Benchmarks
 ==========
 
-For our benchmark we are using a small subset of the dataset `songs.csv`. It was generated with this command:
-```
+## TOC
+
+- [Datasets](#datasets)
+- [Run the benchmarks](#run-the-benchmarks)
+- [Comparison between benchmarks](#comparison-between-benchmarks)
+
+## Datasets
+
+The benchmarks are available for the following datasets:
+- `songs`
+- `wiki`
+
+### Songs
+
+`songs` is a subset of the [`songs.csv` dataset](https://meili-datasets.s3.fr-par.scw.cloud/songs.csv.gz).
+
+It was generated with this command:
+
+```bash
 xsv sample --seed 42 1000000 songs.csv -o smol-songs.csv
 ```
-You can download it [here](https://meili-datasets.s3.fr-par.scw.cloud/benchmarks/smol-songs.csv.gz)
-And the original `songs.csv` dataset is available [here](https://meili-datasets.s3.fr-par.scw.cloud/songs.csv.gz).
 
-We also use a subset of `wikipedia-articles.csv` that was generated with the following command:
-```
+_[Download the generated `songs` dataset](https://meili-datasets.s3.fr-par.scw.cloud/benchmarks/smol-songs.csv.gz)._
+
+### Wiki
+
+`wiki` is a subset of the [`wikipedia-articles.csv` dataset](https://meili-datasets.s3.fr-par.scw.cloud/wikipedia-articles.csv.gz).
+
+It was generated with the following command:
+
+```bash
 xsv sample --seed 42 500000 wikipedia-articles.csv -o smol-wikipedia-articles.csv
 ```
-You can download the original [here](https://meili-datasets.s3.fr-par.scw.cloud/wikipedia-articles.csv.gz) and the subset [here](https://meili-datasets.s3.fr-par.scw.cloud/benchmarks/smol-wikipedia-articles.csv.gz).
 
------
+_[Download the generated `wiki` dataset](https://meili-datasets.s3.fr-par.scw.cloud/benchmarks/smol-wikipedia-articles.csv.gz)._
+
+## Run the benchmarks
+
+### On our private server
+
+The Meili team has self-hosted his own GitHub runner to run benchmarks on our dedicated bare metal server.
+
+To trigger the benchmark workflow:
+- Go to the `Actions` tab of this repository.
+- Select the `Benchmarks` workflow on the left.
+- Click on `Run workflow` in the blue banner.
+- Select the branch on which you want to run the benchmarks and select the dataset you want (default: `songs`).
+- Finally, click on `Run workflow`.
+
+This GitHub workflow will run the benchmarks and push the `critcmp` report to a DigitalOcean Space (= S3).
```

Comment:
Is there a way to know the S3 URL of the critcmp export? Is it displayed in the workflow output?

Reply:
Ok so we can use the `s3cmd ls` command to list the reports and your script to help us compare the results?

Reply:
Yes there is a section `Comparison between benchmarks` at the end of the README (visible in the TOC)

Reply:
And yes, you can see it in the workflow:
<img width="658" alt="Capture d’écran 2021-06-01 à 13 58 54" src="https://user-images.githubusercontent.com/20380692/120319701-95b48d00-c2e1-11eb-9979-b9b5fa68ad15.png">

Reply:
I will update the code so that you can see the file name in the title of the job step

Reply:
And maybe also add this to the little "step-by-step tutorial"? Explain that you can find the name in the job of the workflow :)

Reply:
Why not directly printing the command line that a user can copy/paste to display the diff between the PR branch and master?

Reply:
Hum, the issue is we cannot the name of the file related to the commit on `main`. But I can print a kind of command, yes, it's a good idea